open-sphere-base/core/src/main/java/io/opensphere/core/map/MapManagerMenuProvider.java

```diff
@@ -114,6 +105,64 @@ public int getPriority()
         }
     };
 
+    /**
+     * The popup menu for coordinate information.
+     * 
+     * @param pos the geographic position on the globe.
+     * @return dialog the popup window.
+     */
+    private final JDialog CordMenu(GeographicPosition pos)
```

Comment:
I noticed the right click->open coordinate window is different from the press period coordinate window. These two should be consistent. Also the right click window shouldn't close when you click outside of it.

Reply:
The only difference should be the Title; since it is not supposed to be opened multiple times it did not make sense to keep Mouse Position "#". Is there another difference I am missing?